scripts/patch-release/patch-release.go

```diff
@@ -55,45 +58,71 @@ func run() error {
 		return fmt.Errorf("open local repo: %w", err)
 	}
 
-	currentBranch, err := repo.CurrentBranch()
-	if err != nil {
-		return fmt.Errorf("get current branch: %w", err)
+	for _, minorVersion := range releaseMinorVersions {
+		baseBranchName := branchPrefix + minorVersion // returns "release-x.xx"
+
+		sv, err := getCurrentVersionFromReleaseBranch(repo, baseBranchName) // returns "x.xx.x"
```

Comment:
technically it should be:

```suggestion
		sv, err := getCurrentVersionFromReleaseBranch(repo, baseBranchName) // returns "x.y.z"
```

Reply:
so I do that for this as well?

baseBranchName := branchPrefix + minorVersion // returns "release-x.xx"